### PR TITLE
Finalize car

### PR DESCRIPTION
**Pull request by JakobRoseke**

### CODE REVIEW

File: src/smartCar/smartCar.ino [hunk 1]
```diff
@@ -55,46 +52,51 @@ void setup()
...
-
+    
     //Setup the webserver
     webserverInit();
 }
 
 void loop()
 {
     car.update();
+    gyroscope.update();
```
Comment (platisd) [inline src/smartCar/smartCar.ino]: `car.update()` already updates the gyroscope.

File: src/smartCar/smartCar.ino [hunk 2]
```diff
@@ -55,46 +52,51 @@ void setup()
...
+    
+    //Initialize the pin
     pinMode(LED_BUILTIN, OUTPUT);
     
     //Update the webserver
     webserverCreation();
 
-    checkDestination();
+    //Check if destination has been reached
+    //checkDestination();
```
Comment (platisd) [inline src/smartCar/smartCar.ino]: Should this be commented and not just removed?

File: src/smartCar/smartCar.ino [hunk 3]
```diff
@@ -298,13 +300,16 @@ String sendHTML(char message) {
   return html;
 }
 
-void checkDestination(){
+
+//This code should have worked in theory
+//But the gyro heading did not function properly and we ran out of time to fix it.
+/*void checkDestination(){
```
Comment (platisd) [inline src/smartCar/smartCar.ino]: If it doesn't work, then why keep it? :)

**Comment by Rilindzuta** [comment]
Nice

**Review by Rilindzuta — APPROVED** [review]
Nice

**Pull request merged**